src/control/cmd/dmg/main.go

```diff
@@ -82,37 +79,89 @@ func appSetup(broadcast bool) error {
 	return nil
 }
 
-func main() {
-	if dmgMain() != nil {
-		os.Exit(1)
-	}
+type broadcastCmd struct{}
+
+func (broadcastCmd) isBroadcast() {}
+
+type broadcaster interface {
```

Comment:
why put the interface definition after the Interface implementations? I'm used to the other way round.

Reply:
No particular reason; that's just how it was written. In Go it doesn't matter. I'm happy to put it the other way around if that looks better.

Reply:
no problem, was just curious